### PR TITLE
Home page, Header and Footer

### PR DESCRIPTION
**Pull request by TomDijkema**
PR adds content to the Home Page based upon the design made for the Marketplace.
This includes: a title, search bar and search categories on the bottom of the page.
The search bar will redirect the user's query to the future search page.

PR also adds generic Header and Footer components

### CODE REVIEW

File: src/components/general/footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+/* Import Styles */
+import styles from './footer.module.scss';
+
+
+/** Component that renders the application's Footer */
+const Footer = () => {
+    const currentYear = new Date().getFullYear();
+
+    return (
+        <Container fluid>
```
Comment (southeo) [inline src/components/general/footer/Footer.tsx:16]: Why does this look like html in a .tsx file? I know we can put javascript/css in an html file, but that's bad practice (as far as i know). 

Thanks for clarifying!

File: src/webroot/img/cetafLogo.png
Comment (southeo) [inline src/webroot/img/cetafLogo.png:1]: honey i shrunk the cetaf

File: src/webroot/img/tettrisBG.jpg
Comment (southeo) [inline src/webroot/img/tettrisBG.jpg:1]: duplicate?

**Review by southeo — APPROVED** [review]
1 clarification, 1 comment, and 1 joke

File: src/webroot/img/tettrisBG.jpg
Comment (TomDijkema) [reply src/webroot/img/tettrisBG.jpg:1]: It is, removed it

File: src/webroot/img/cetafLogo.png
Comment (TomDijkema) [reply src/webroot/img/cetafLogo.png:1]: Part 4 ;)

File: src/components/general/footer/Footer.tsx (hunk 1 above)
Comment (TomDijkema) [reply src/components/general/footer/Footer.tsx:16]: This is the cool thing of React, you are allowed to put HTML elements in there and a bunch of them are actually called JSX components. A JSX component like the home page itself (TSX is JSX with typing) is a component, and a component consists of HTML, Javascript, CSS, other components, you name it. Functional stuff in JS is in the top of the component, I can talk days about the hierarchy and how this all works, but to understand: the global return statement in TSX file returns an HTML template.

In regular HTML files, it is very basic to include JS and CSS, it is allowed but makes the files unnecessarily complicated and long. That is why we split it up into an HTML file, JS file and CSS fille. We still do this with TSX files, but I have loose CSS files to import. JS is like I said includes in TSX files, but if it gets too global or much I tend to take it out and make a TS file (without the x). A TS file only allows for JS and cannot return HTML templates.

I can give you an better explanation about this on Thursday or some other time as its a fundamental of the JS framework we use, aka React.

Comment (southeo) [reply src/components/general/footer/Footer.tsx:16]: hell ya, sounds good

**Pull request merged**